src/khoj/routers/helpers.py

```diff
@@ -287,8 +287,8 @@ async def aget_relevant_information_sources(
 
     try:
         response = response.strip()
-        # Remove any markdown json codeblock formatting if present (useful for gemma-2)
-        if response.startswith("```json"):
+        # Remove any markdown json codeblock formatting if present. Useful for non schema enforceable models
+        if response.startswith("```json") and response.endswith("```"):
```

Comment:
Future optimization, maybe we can move this custom logic to a helper method for code re-use/maintainability.

Reply:
Good point. Put that json codeblock removal code snippet under conversation utils as a helper function for reusability